aria-practices.html

```diff
@@ -2150,6 +2150,43 @@ <h4>WAI-ARIA Roles, States, and Properties</h4>
       </section>
     </section>
 
+    <section class="widget" id="meter">
+      <h3>Meter</h3>
+      <p>
+        A <a class="role-reference" href="#meter">meter</a> allows authors to define a value within a known range. The minimum and maximum limits of the range must be defined numerically, and the value should be a number within the range.
+      </p>
+      <p>
+        Meter examples could include current battery percentage, or a car's fuel level. A <code>meter</code> could not be used to represent a value like the current world population, since there is no meaningful maximum limit.
```

Comment:
> Meter examples could include current battery percentage, or a car's fuel level

Propose to change this to:

> Meter example include a device’s current batter percentage and a car’s fuel level.

---

> A <code>meter</code> could not be used to

Propose we change to either:

> A <code>meter</code> cannot be used to…

…or…

> A <code>meter</code> should not be used to…

I think the first one is a little more strict and to the point. But the second one reads a little bit friendlier (and is more consistent with other text). Thoughts?

Reply:
I like should, friendly sounds like a good approach :)